cluster/aws/util.sh

```diff
@@ -1096,8 +1184,13 @@ function wait-master() {
   echo "  up."
   echo
 
+  local url="https://${KUBE_MASTER_IP}/healthz"
+  if [[ -n "${KUBE_SERVER:-}" ]]; then
+    url="${KUBE_SERVER}/healthz"
+  fi
+
```

Comment:
I've added some notes to cluster/common.sh for KUBE_MASTER_IP and KUBE_SERVER, and to cluster/aws/util.sh for API_SERVERS and KUBE_MASTER_DNSNAME